jaxlib/mosaic/gpu/custom_call.cc

```diff
@@ -565,7 +644,7 @@ void** MosaicGpuCompile(const char* module) {
   if (!compiled.ok()) {
     return nullptr;
   }
-  auto [ctx, launch] = compiled->GetHostLaunch();
+  auto [ctx, launch, is_comm_used] = compiled->GetHostLaunch();
```

Comment:
With this PR we won't be able to use those comms kernels with our PyTorch compilation path, but I guess that's ok for now